vmdb/app/controllers/api_controller.rb

```diff
@@ -95,6 +96,10 @@ def initialize
     @api_token_mgr   = TokenManager.new(@module)
   end
 
+  def self.all_encrypted_attributes
+    (ATTR_TYPES[:encrypted] | ::Vmdb::ConfigurationEncoder::PASSWORD_FIELDS).sort
```

Comment:
Why create a method when this can just live in ATTR_TYPES directly.  On that note, why is ConfigurationEncoder involved at all?  Those PASSWORD_FIELDS are embedded into a configuration yaml, which I doubt you're ever going to return over the REST API.  None of them are columns, so it's not necessary.